### DISCUSSION TITLE
Blog: Announcing SixtyFPS 0.1

### DISCUSSION THREAD

**Discussion by ogoffart** [slint-ui/slint#420] [Blog Comments]
This is the discussion thread for the blog post [Announcing SixtyFPS 0.1](https://sixtyfps.io/blog/announcing-sixtyfps-0.1.html)

https://sixtyfps.io/blog/announcing-sixtyfps-0.1.html

**Comment by Be-ing** [comment]
Is there some documentation how the Qt integration works? Does Qt need to be linked at compile time? Could an application be built without Qt present and dynamically load Qt at runtime if available?

**Reply by ogoffart** [reply]
As of now, the decision to use Qt and the native style is a compile time decision.
